debian/copyright

```diff
@@ -1,22 +1,381 @@
-This is syslog-ng, written and maintained by  <bazsi@balabit.hu>
-on Sat, 03 Mar 2007 16:07:59 +0100
-
-The original source can always be found at:
-	http://www.balabit.hu/downloads/files/syslog-ng
-
-Copyright (C) 2002-2011 BalaBit IT Security Ltd
-Copyright (C) 1998-2011 Balazs Scheidler
-
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: syslog-ng
+Upstream-Contact: Syslog-ng users' and developers' mailing list <syslog-ng@lists.balabit.hu>
+Source: git://github.com/balabit/syslog-ng.git
+Copyright: Copyright (C) Balázs Scheidler
+           Copyright (C) BalaBit IT Security Ltd.
+
```

Comment:
I realised that this lists all copyrights, any chance of using tests/copyright/policy somehow? Do we need this specific format?

I wouldn't stop integrating this patch because of this, but maybe drop this file until this one is covered.